doc/2/guides/getting-started/5-realtime-engine/index.md

```diff
@@ -0,0 +1,188 @@
+---
+code: false
+type: page
+title: Realtime Engine
+description: Subscribe to realtime notifications from the database
+order: 500
+---
+
+# Realtime Engine
+
+Kuzzle integrates an advanced realtime engine. It can work in a classic pub/sub mode but also as a realtime database notification engine.
+
+**Any change occuring on the database can generate a realtime notification** about the creation, modification and deletion of documents.
+
+A filter system also allows clients to receive notifications only according to their interest in certain types of documents.
+
+Subscription to realtime notifications is client-side and does not require additional server-side code.
+
+### Subscribe to changes
+
+First, we need to subscribe to changes occuring in a specific collection:
+
+```bash
+$ kourou realtime:subscribe nyc-open-data yellow-taxi
+```
+
+::: info
+This will use the [realtime:subscribe](/core/2/api/controllers/realtime/subscribe) API action. 
+::: 
+
+Kourou is now waiting for realtime notifications about changes in our collection.
+
+We are going to create a new document in the Admin Console to trigger one!
+
+Open the [Admin Console](http://console.kuzzle.io/#/data/nyc-open-data/yellow-taxi) and then create a new document. 
+
+Then you should receive a notification about the created document in your terminal:
+
+```bash
+ 🚀 Kourou - Subscribes to realtime notifications
+ 
+ [ℹ] Connecting to ws://localhost:7512 ...
+ [ℹ] Waiting for realtime notifications on "nyc-open-data":"yellow-taxi" ...
+
+ [ℹ] New notification triggered by API action "document:create"
+ {
+  "_id": "dvIdJ3UB1MqPtuLKxKDS",
+  "_source": {
+    "name": "Aschen",
+    "city": "Antalya",
+    "age": 27,
+    "_kuzzle_info": {
+      "author": "-1",
+      "createdAt": 1602679063751,
+      "updatedAt": null,
+      "updater": null
+    }
+  },
+  "_version": 1
+}
+```
+
+### Use subscription filters
+
+Kuzzle allows you to use filters to receive only the notifications matching specific types of documents.  
+
+Again, we are going to subscribe to notifications but only for documents where:
+  - `age` is greater than `30`
+  - `city` is equal to `Antalya`
+
+This filter must be written using Koncorde Filter DSL, which is a little inspired by Elasticsearch DSL:
```

Comment:
TODO: add a link to the Koncorde DSL handbook (when it will be ready). How do we express such reminders?

Reply:
We just add links who doesn't exists yet like `[Koncorde Filter DSL](/core/2/some-link)`